### DISCUSSION TITLE
What is Netdata way to monitor periodic tasks like cron jobs? Is Prometheus Pushgateway required?

### DISCUSSION THREAD

**Discussion by powerman** [netdata/netdata#17905] [Q&A]
I'd like to monitor result of daily backup task and get alerts if it fail.

Should I install Pushgateway on every system (they all do own backups) side-by-side with Netdata to make this work, or there are other ways to implement this?
Are there any hints how to setup Pushgateway and design backup job metrics to make it easier to consume by Netdata?

Also, is there a way to somehow turn unattended-upgrades email reports into Netdata metrics/alert?

**Comment by dlidstrom** [comment]
Did you figure out a way to monitor jobs?
